backend/app/services/answer_generator.py

```
@@ -214,7 +221,14 @@ def _build_context(
 
                 for idx, a in enumerate(alert_summaries[:10], start=1):
                     cam_str = f"Camera {a['camera_id']}" if a.get("camera_id") is not None else "Unknown camera"
-                    prompt += f"- {idx}. [{a.get('severity', 'info')}] {a.get('alert_name') or 'Unnamed alert'} at {a.get('triggered_at') or 'unknown time'} on {cam_str}: {a.get('message')}\n"  # noqa: E501
+                    trigger_ts = a.get('triggered_at')
+                    try:
+                        if trigger_ts:
+                            dt = datetime.fromisoformat(str(trigger_ts).replace('Z', '+00:00'))
+                            trigger_ts = dt.strftime("%Y-%m-%d %I:%M:%S %p")
+                    except Exception:
+                        pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)